app/views/content_items/consultation.html.erb

```diff
@@ -154,43 +154,40 @@
     <% if @content_item.ways_to_respond? %>
       <div id="ways-to-respond" class="consultation-ways-to-respond">
         <%= render 'govuk_publishing_components/components/heading', text: t("consultation.ways_to_respond"), mobile_top_margin: true %>
-          <% @ways_to_respond_body = capture do %>
-            <% if @content_item.respond_online_url %>
-              <div class="call-to-action">
-                <p><%= link_to t("consultation.respond_online"), @content_item.respond_online_url %></p>
-              </div>
 
-              <% if @content_item.email || @content_item.postal_address %>
-                <p>or</p>
-              <% end %>
-            <% end %>
+        <%= render 'govuk_publishing_components/components/govspeak', {
+          direction: page_text_direction,
+        } do %>
+          <% if @content_item.respond_online_url %>
```

Comment:
This is much nicer than using a `capture` block. It's much clearer because it keeps everything together.